core/docker/docker-compose.config.yml

```diff
@@ -3,10 +3,12 @@ version: '3.3'
 services:
   # container for executing config import scripts for setting up initial config
   config-importer:
-    container_name: openhim-config-importer
+    container_name: instantohie-config-importer
```

Comment:
We might need to revert this back to be specific to the package creating this container.
If we have another package also making use of this image for implementing config we wont be able to use the same container name. This way we also keep it specific to the package its performed on